nuxeo-java-client/src/main/java/org/nuxeo/client/util/Base64.java

```diff
@@ -216,6 +216,10 @@ public static String encode(String content) {
         return encodeBytes(content.getBytes());
     }
 
+    public static String encode(String content, int options) {
```

Comment:
It might be clear if you can add Javadoc to explain the parameter `options`, something like:

```java
/**
 * @params options encoding options provided by {@link Base64}.
 * @since 3.0.0
 * @see Base64#DONT_BREAK_LINES
 */
public static String encode(String content, int options) { ... }

Reply:
It might be better to use an existing library instead of reinventing the wheel 😈